pkg/scheduler/algorithm/well_known_labels.go

```diff
@@ -61,4 +61,9 @@ const (
 	// from the cloud-controller-manager intitializes this node, and then removes
 	// the taint
 	TaintExternalCloudProvider = "node.cloudprovider.kubernetes.io/uninitialized"
-)
+
+	// When node is shutdown in external cloud provider
+	// stopped flag is needed for immediately volume detach
+	// after node comes back, the taint is removed
+	TaintNodeStopped = "node.cloudprovider.kubernetes.io/stopped"
```

Comment:
We should choose between shutdown or stopped to be consistent

Reply:
maybe shutdown is better word for this